source/config.js

```diff
@@ -1,4 +1,18 @@
+import { z } from "zod"
+
+export const schema = z.object({
```

Comment:
look forward to playing around with this.  -- and i think that this is going to be the way to go ...

but I do want this to be configurable by customers.

but i think that we can do something simple in the 

https://github.com/jtushman/babar/blob/jt/configuration/examples/.babarrc.json

and in the config.js file

in which we can just have the list the sections with the descriptions.